src/conversion/buildviewconverter.js

```diff
@@ -141,11 +141,21 @@ class ViewConverterBuilder {
 	 * @param {String|RegExp} [value] View attribute value.
 	 * @returns {module:engine/conversion/buildviewconverter~ViewConverterBuilder}
 	 */
-	fromAttribute( key, value = /.*/ ) {
+	fromAttribute( key, value = /.+/ ) {
```

Comment:
Why this change? There are empty attributes which values can be empty but still they "are".